### PR TITLE
added scripts for deployment with an existing tokens registry

### PR DESCRIPTION
**Pull request by kate-aleksseeva**

### CODE REVIEW

File: scripts/create_single_recipient_setup.py [hunk 1]
```diff
@@ -29,16 +33,22 @@ def main():
...
 
-    log.ok("Tokens", deploy_config.tokens)
-    log.ok("Title", deploy_config.title)
-    log.ok("Trusted caller", deploy_config.trusted_caller)
     log.ok("Limit", deploy_config.limit)
     log.ok("Period", deploy_config.period)
     log.ok("Spent amount", deploy_config.spent_amount)
-
+    log.ok("Title", deploy_config.title)
+    log.ok("Trusted caller/recipient", deploy_config.trusted_caller)
```
Comment (zuzueeka) [inline scripts/create_single_recipient_setup.py:51]: remove recipient

File: scripts/create_single_recipient_setup.py [hunk 2]
```diff
@@ -11,16 +11,20 @@
 from brownie import AllowedRecipientsBuilder
```
Comment (zuzueeka) [inline scripts/create_single_recipient_setup.py:12]: please add comment on how to use tokens and tokens registry params
and for grant_rights param

File: scripts/create_full_recipients_setup.py [hunk 3]
```diff
@@ -15,18 +15,50 @@
...
+    trusted_caller="",
+    grant_rights = False, # permissions to execute AddAllowedRecipient / RemoveAllowedRecipient methods on behalf of trusted_caller
 )
 
 
 def main():
+
+    new_token_registry_is_required = not bool(deploy_config.tokens_registry)
+
+    if new_token_registry_is_required and not bool(deploy_config.tokens):
```
Comment (rkolpakov) [inline scripts/create_full_recipients_setup.py:58]: I would add a check that both fields tokens and tokens_registry are filled to avoid misbehaviour

**Review by rkolpakov — APPROVED** [review]

**Comment by zuzueeka** [comment]
lgtm

**Review by zuzueeka — APPROVED** [review]
lgtm

**Pull request merged**